library_generation/test/resources/google-cloud-java_config.yaml

```diff
@@ -0,0 +1,59 @@
+#Required.
+gapic_generator_version: 2.30.0
+#Required.
+googleapis-commitish: 4512234113a18c1fda1fb0d0ceac8f4b4efe9801
+#Required.
+owlbot-cli-image: sha256:623647ee79ac605858d09e60c1382a716c125fb776f69301b72de1cd35d49409
+#Required.
+synthtool-commitish: 59fe44fde9866a26e7ee4e4450fd79f67f8cf599
+#Required.
+python-version: 3.11.2
+#Optional. The root folder name of generated client libraries. If empty, modules will be created under current folder, useful for single module
+destination-path: google-cloud-java
+#Required.
+apis:
+    #Required. The folder name of the module. This is the destination-name in new-client.py.
+  - name: java-asset
+    #Optional. The default value is the title of service yaml
+    name-pretty: Cloud Asset
+    #Required.
+    api_shortname: asset
+    #Required.
+    library_type: GAPIC_AUTO
+    #Optional. The default value is com.google.cloud
+    group_id: com.google.cloud
+    #Optional. The default value is google.cloud.{api_shortname}
+    artifact_id: google.cloud.asset
+    #Optiona. The default value is true.
+    requires-billing: true
+    #Optional. The default value is documentation.summary from service yaml
+    api_description:
+    #Optional.
+    product_documentation:
+    #Optional.
+    client_documentation:
+    #Optional.
+    rest_documentation:
+    #Optional.
+    rpc_documentation:
+    #Required.
+    services:
+        #Required. relative path to google3/third_party/googleapis/stable
+      - proto_path: google/cloud/asset/v1
+        #Required. Remove this parameter once https://github.com/googleapis/sdk-platform-java/issues/2050 is fixed.
+        proto_only: false
```

Comment:
`proto_only` is parsed from BUILD.bazel, do you want to add this parameter here?

Reply:
We don't need it if we can parse it, I thought we have to specify it. How do we parse it?

Reply:
The logic is if we can find `java_gapic_library` target in BUILD, then set the value to `true`; otherwise set to `false`.

Reply:
Sounds god. I'll remove it.